charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-saved-objects-config.yaml

```diff
@@ -0,0 +1,107 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kibana-saved-objects
+  namespace: {{ .Release.Namespace }}
+data:
+  index-pattern.json: |
+    {"attributes": {"title": "logstash-*", "timeFieldName": "@timestamp"}}
+  dashboard.json: |
+{
```

Comment:
error converting YAML to JSON: yaml: line 10: could not find expected ':'. You could fix the yaml.